priv/repo/migrations/20160328115654_create_relays.exs

```diff
@@ -7,7 +7,7 @@ defmodule Cog.Repo.Migrations.AddRelays do
       add :name, :text, null: false
       add :token_digest, :text, null: false
       add :enabled, :boolean, null: false
-      add :desc, :text, null: true
```

Comment:
Oh yeah good point about the release. I'd probably bias towards a new migration but this is probably fine since no one has run it yet. Whichever is good.